java/component/keyple-plugin/pcsc/src/main/java/org/eclipse/keyple/plugin/pcsc/PcscReaderSetting.java

```diff
@@ -0,0 +1,56 @@
+/* **************************************************************************************
+ * Copyright (c) 2020 Calypso Networks Association https://www.calypsonet-asso.org/
+ *
+ * See the NOTICE file(s) distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License 2.0 which is available at http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ ************************************************************************************** */
+package org.eclipse.keyple.plugin.pcsc;
+
+/**
+ * This class defines the constants useful for the configuration of the PC/SC readers.<br>
+ * These constants are strings defining the keys (prefix KEY_) and values to be used with the
+ * setParameter method.
+ */
+public final class PcscReaderSetting {
+  /** key to set the transmission mode */
+  public static final String KEY_TRANSMISSION_MODE = "transmission_mode";
```

Comment:
PcscReaderSetting is a public final class: all its public elements are static final attributes.
These parameters could still be located in the PcscReader interface.

Reply:
The idea is to answer this point raised by SonarQube:
https://sonarcloud.io/project/issues?id=eclipse_keyple-plugin_pcsc&open=AXMjT1DptNti-jTiWO4G&resolved=false&severities=CRITICAL&types=CODE_SMELL
The constant interface pattern is generally discouraged.

Reply:
@andrei-cristea , do you have an opinion or advice on this subject?

Reply:
I agree with Sonar